src/lib.rs

```diff
@@ -28,6 +28,8 @@
 //! * [`ArrayVec`](ArrayVec) is an array-backed vec-like structure with a fixed
 //!   capacity. If you try to grow the length past the array's capacity it will
 //!   error or panic (depending on the method used).
+//! * [`SliceVec`](SliceVec) is similar, but instead of the vec having an owned
+//!   array, it holds onto a unique borrow of a slice.
```

Comment:
SliceVec has an advantage of being cheap to pass around. Passing an array (and by extension ArrayVec) is expensive because it moves the value and copies the entire thing, but SliceVec does not suffer from this issue. Perhaps we should mention it in the docs.

That was brought up here: https://www.reddit.com/r/rust/comments/fshuhk/introducing_tinyvec_100_safe_alternative_to/fm4cd9a/

Reply:
In fact, ArrayVec is starting to look a little redundant because you can always slice an array and make a SliceVec out of it

Reply:
`ArrayVec: 'static` while `SliceVec` is not. For the linked comment another option would be to pass around an `&mut TinyVec` where possible rather than move it everywhere.